addons/mail/static/src/js/utils.js

```diff
@@ -95,6 +95,7 @@ function htmlToTextContentInline(htmlString) {
     const fragment = document.createDocumentFragment();
     const div = document.createElement('div');
     fragment.appendChild(div);
+    htmlString = htmlString.replace(/<br\s*\/?>/gi,' ');
```

Comment:
I think this is safe... But just to make sure, do you think it's possible to have sanitized message body which, when replacing `<br/>`s, is prone to injections?

Reply:
In fact even without br's I wonder how it could not be a problem .. When replacing there is a problem with a not valid tag... 


IN : <p>&lt;script&gt;throw new Exception("OUHLA !");&lt;/script&gt;</p>
OUT : <script>throw new Exception("OUHLA !");</script>

and it's used in a `t-raw` which does not display anything (but it should display <script ... IMO) but the exception is not thrown

Reply:
I'd add
```
.replace('<', '&lt;')
.replace('>', '&gt;')
```
at the end, what do you think ?

Reply:
Was able to trigger an `alert()` on opening messaging menu. Had to post this as last message of a channel:
```
<p><script>alert(0);</script></p>
```

Reply:
Good catch, glad we resolved this now.